yt/geometry/coordinates/geographic_coordinates.py

```diff
@@ -168,11 +168,33 @@ def pixelize(self, dimension, data_source, field, bounds, size,
             return self._cyl_pixelize(data_source, field, bounds, size,
                                           antialias, dimension)
         elif self.axis_name[dimension] == self.radial_axis:
-            return self._ortho_pixelize(data_source, field, bounds, size,
+            return self._cartesian_pixelize(data_source, field, bounds, size,
                                         antialias, dimension, periodic)
         else:
             raise NotImplementedError
 
+    def _cartesian_pixelize(self, data_source, field, bounds, size, antialias,
+                            dimension, periodic):
+
+        period = self.period[:2].copy()
+        period[0] = self.period[self.x_axis[dimension]]
+        period[1] = self.period[self.y_axis[dimension]]
+        if hasattr(period, 'in_units'):
+            period = period.in_units("code_length").d
+
+        # For a radial axis, px will correspond to longitude and py will
+        # correspond to latitude.
+        px = data_source["px"]
+        pdx = data_source["pdx"]
+        py = data_source["py"]
+        pdy = data_source["pdy"]
+        buff = np.zeros((size[1], size[0]), dtype="f8")
+        pixelize_cartesian(buff, px, py, pdx, pdy,
+                           data_source[field],
+                           bounds, int(antialias),
+                           period, int(periodic))
+        return buff
+
     def _ortho_pixelize(self, data_source, field, bounds, size, antialias,
```

Comment:
Are you still using this one? If not, could we remove it?

Reply:
Ah good catch. I meant to remove that.